### PR TITLE
AM-2877: Tidy up logging in topic consumer classes

### PR DESCRIPTION
**Pull request by tomxelliott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-2877

### Change description ###
Tidy up logging in topic consumer classes

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/servicebus/CRDTopicConsumerTest.java [hunk 1]
```diff
@@ -39,19 +50,41 @@ void getSubscriptionClientThrowsServiceBusException() {
...
-
     @Test
     void registerMessageHandlerOnClientTest() throws Exception {
-        CompletableFuture<Void> voidCompletableFuture = sut.registerCRDMessageHandlerOnClient(
-                subscriptionClient);
+        CompletableFuture<Void> voidCompletableFuture = sut.registerCRDMessageHandlerOnClient(subscriptionClient);
         assertNull(voidCompletableFuture);
     }
 
+    @Test
```
[the remark this answers is not in the document]
Comment (mattnayler) [reply src/test/java/uk/gov/hmcts/reform/orgrolemapping/servicebus/CRDTopicConsumerTest.java:64]: Resolved in 5bd7ac3be753a03ac8fb45512ad981ee01be3568.

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/servicebus/JRDTopicConsumerTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    void shouldThrowExceptionWhenCallToRoleAssignmentServiceIsUnsuccessful() throws JsonProcessingException {
+
+        UserRequest userRequest = UserRequest.builder()
+                .userIds(List.of("9d6089ab-0459-4bb6-9174-dc111f0f661d", "1ab7da58-fa93-43b7-8df1-8f8406fdcb5a"))
+                .build();
+        ObjectMapper mapper = new ObjectMapper();
+        String request = mapper.writeValueAsString(userRequest);
+
+        when(deserializer.deserialize(any())).thenReturn(userRequest);
+        when(bulkAssignmentOrchestrator.createBulkAssignmentsRequest(userRequest, UserType.CASEWORKER))
```
Comment (mattnayler) [inline src/test/java/uk/gov/hmcts/reform/orgrolemapping/servicebus/JRDTopicConsumerTest.java:75]: This should be passing `UserType.JUDICIAL`

**Review by mattnayler — APPROVED** [review]

**Review by prashanthkumarnb — DISMISSED** [review]

**Review by mattnayler — APPROVED** [review]
Re-approved at 1a839c62a2d3740d333f1eb45f70e967adca2318.

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Pull request merged**